bridge-patterns/bridge-frontend
# Feature join bridge platform

- What does this PR do?
This PR introduce the JOIN feature to the Bridge website

- What Trello task is related to this PR?
The Trello card is[ here](https://trello.com/c/2zkPKUtX)

- How do I test out this PR?
Git clone this repo `cd bridge-frontend/bridge` Then run `npm install` to get all the necessary dependencies. Afterwards run `npm start` to test out the dashboards accordingly.
![image](https://user-images.githubusercontent.com/1590213/84558740-ab7e1a80-ad3d-11ea-8c92-d15309723bb2.png)



- Which end points should are ready for testing?
There are no endpoints for testing.
